### DISCUSSION TITLE
Icon on Flatpak has no white background

### DISCUSSION THREAD

**Discussion by xlionjuan** [rustdesk/rustdesk#7501] [General]
So it will looks too dark sometimes

![Screenshot 2024-03-24 21 06 43](https://github.com/rustdesk/rustdesk/assets/25243894/63d54f65-56d8-442f-ab24-7e46bcd9c7dc)

**Comment by rustdesk** [comment]
https://github.com/rustdesk/rustdesk/commit/09ec0b63258562f484deab54ee3169dae8f78124
